modules/configuring_hyperthreading_for_a_cluster.adoc

```diff
@@ -83,20 +84,20 @@ The reserved and isolated CPU pools must not overlap and together must span all
 
 [IMPORTANT]
 ====
-Hyperthreading is enabled by default on most Intel processors. If you enable hyperthreading, all threads processed by a particular core must be isolated or processed on the same core.
+Hyper-Threading is enabled by default on most Intel processors. If you enable Hyper-Threading, all threads processed by a particular core must be isolated or processed on the same core.
 ====
```

Comment:
Let's also note that when HT is enabled, all guaranteed pods must use multiples of the SMT level to avoid a noisy neighbor situation (in which the pod will fail with a relevant error). SMT is on when there are two threads per core; hence, the requested CPUs must be an even number.

Reply:
Documented upstream - see full-pcpus-only here https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy-options